cmd/devp2p/internal/ethtest/suite_test.go

```diff
@@ -52,6 +52,7 @@ func TestEthSuite(t *testing.T) {
 				t.Fatal()
 			}
 		})
+		time.Sleep(100 * time.Millisecond)
```

Comment:
This is a bit indicative of it still being flaky. I'm guessing you added it to make the spurious packets from earlier tests to interfere with later tests, but is it still needed? 
If it's only once in a while, maybe we can either figure out what spurious packets those are, and just 'continue' on them, _or_ maybe somehow drain the deliver buffer between test executions? 

Just having a random 100ms timout like that means it'll work "more often", but still spuriously fail on e.g. appveyor. 

Not a blocker, but I would prefer if we solved it without having that in

Reply:
I agree.